Sources/PostgreSQL/Connection/PostgreSQLConnection+ServerAddress.swift

```diff
@@ -11,6 +11,16 @@ extension PostgreSQLConnection {
             return .unixSocket(path: "/tmp/.s.PGSQL.5432")
         }
         
+        /// TCP or Unix socket PostgreSQL address depending on the host format.
+        public init(hostname: String, port: Int) {
```

Comment:
nit: how about `static func tcpOrSocket` instead? It feels weird to have one initializer as `init` and the other ones as `.tcp`, `.unixSocket`, etc.

Reply:
I agree. I'll change that.